Common/tests/GrpcHealthChecksServiceTest.cs

```diff
@@ -0,0 +1,51 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2023. All rights reserved.
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Threading.Tasks;
+
+using ArmoniK.Api.gRPC.V1.HealthChecks;
+using ArmoniK.Core.Common.gRPC.Services;
+using ArmoniK.Core.Common.Tests.Helpers;
+
+using NUnit.Framework;
+
+namespace ArmoniK.Core.Common.Tests;
+
+[TestFixture]
+public class GrpcHealthChecksServiceTest
+{
+  [Test]
+  public async Task CheckHealthShouldSucceed()
+  {
+    var taskTable     = new SimpleTaskTable();
+    var objectStorage = new SimpleObjectStorage();
+    var queueStorage  = new SimplePushQueueStorage();
```

Comment:
I think it would be better to have `Mock<>` and ensure that `Check` is actually called on those mocks.